package.json

```diff
@@ -62,37 +62,37 @@
     "test:types": "pnpm dev:prepare && tsc --noEmit && nuxi typecheck playground"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.15.1",
```

Comment:
@ineshbose / @marr Moving this from a dependency to a dev dependency broke the package for us in: https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com/-/merge_requests/4338 because nuxt@2 doesn't seem to ship `@nuxt/kit` per default. 

It probably should be a peer dependency at least

Reply:
`@nuxt/kit` should be an explicit dependency - I did not spot that this PR moved it.

unrelatedly, do you need any help migrating https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com to nuxt 3?